### ISSUE TITLE
get-pip no longer runs on Python2

### ISSUE THREAD

**Issue by dogtopus** [renpy/renpy-build#14]
The deprecation is on schedule for quite a while now so I'm not surprised it finally stopped working.

```
Traceback (most recent call last):
  File "/opt/renpy-build/tmp/get-pip.py", line 24244, in <module>
    main()
  File "/opt/renpy-build/tmp/get-pip.py", line 199, in main
    bootstrap(tmpdir=tmpdir)
  File "/opt/renpy-build/tmp/get-pip.py", line 82, in bootstrap
    from pip._internal.cli.main import main as pip_entry_point
  File "/tmp/tmpeSU33p/pip.zip/pip/_internal/cli/main.py", line 60
    sys.stderr.write(f"ERROR: {exc}")
                                   ^
SyntaxError: invalid syntax
```

Also see https://pip.pypa.io/en/stable/development/release-process/#python-2-support

**Comment by dogtopus** [comment]
A quick workarond before a properfix would be using an old get-pip and installing the same old pip with it.

For example, 

```sh
if [ ! -e $GET_PIP ]; then
    curl https://web.archive.org/web/20210115074757id_/https://bootstrap.pypa.io/get-pip.py -o $GET_PIP
    sudo python2 $GET_PIP 'pip<21'
fi
```

(This works on my machine)

**Comment by scj643** [comment]
This has been fixed with #16
